### ISSUE TITLE
load testset 

### ISSUE THREAD

**Issue by herochen7372** [wholebody3d/wholebody3d#32]
Hi, Could you help me? The previous work used to use the Human36M dataset S9, S11 as a test set, however, the test set you provided is S8. And his annotation format is not quite the same as the training set, can you provide the code for loading the test set data？
